docs/extend/extend-apiml/onboard-direct-eureka-call.md

```diff
@@ -218,6 +218,18 @@ The following parameters define service information for the API Catalog:
 
     **Important!**  When the expected encoded character is an encoded slash or backslash (`%2F`, `%5C`), make sure the Gateway is also configured to allow encoded slashes. For more info see [Installing the Zowe runtime on z/OS](../../user-guide/install-zos.md).
 
+* **apiml.connectTimeout**
+    
+    Value in milliseconds that specifies a period, in which this service should establish a single non-managed connection with API ML. If omitted, default value specified in API ML gateway service configuration is used.
```

Comment:
Shouldn't it be the other way round?

Value in milliseconds that specifies a period, in which the API ML should establish the connection with this service.